src/modalities/config/config.py

```diff
@@ -246,6 +249,38 @@ class LLMDataLoaderConfig(BaseModel):
     skip_num_batches: Optional[int] = 0
 
 
+class HuggingFaceAdapterConfig(ABC, PretrainedConfig):
```

Comment:
Consider moving the adapter configs into their own file. This file is already pretty long and the adapter configs are not strongly coupled with the other configs.

Reply:
We actually had these in different files, but were asked to merge them together with the other configs. So I would keep it here